cogboard-webapp/src/components/MainTemplate/index.js

```diff
@@ -27,7 +27,10 @@ import {
 } from './styled';
 
 const MainTemplate = () => {
-  const currentBoardId = useSelector(({ ui }) => ui.currentBoard);
+  const currentBoardId = useSelector(({ ui }) => ui.currentBoard) || '';
+  const currentBoardType = useSelector(
```

Comment:
It is not a type rather whole board. Type is a string not an object representing board.

Reply:
Done